src/domain/community/contributor/organization/context/OrganizationInfoFragment.graphql

```diff
@@ -17,7 +17,7 @@ fragment OrganizationInfo on Organization {
     displayName
     description
     visual(type: AVATAR) {
-      ...VisualUri
+      ...VisualFull
```

Comment:
VisualFull has a lot of things, not only the alt text, are you sure we need all of that when we were only requesting id uri and name before?